tests/cnf/ran/ptp/internal/iface/helper.go

```diff
@@ -0,0 +1,116 @@
+package iface
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/rh-ecosystem-edge/eco-goinfra/pkg/clients"
+	"github.com/rh-ecosystem-edge/eco-gotests/tests/cnf/ran/ptp/internal/ptpdaemon"
+)
+
+// GetNICDriver uses ethtool to retrieve the driver for a given network interface on a specified node.
+func GetNICDriver(client *clients.Settings, nodeName string, ifName Name) (string, error) {
+	command := fmt.Sprintf("ethtool -i %s | grep --color=no driver | awk '{print $2}'", ifName)
+	output, err := ptpdaemon.ExecuteCommandInPtpDaemonPod(client, nodeName, command)
+
+	if err != nil {
+		return "", fmt.Errorf("failed to get NIC driver for interface %s on node %s: %w", ifName, nodeName, err)
+	}
+
+	return strings.TrimSpace(output), nil
+}
+
+// GetOCPInterfaceName retrieves the name of the interface that is connected to the OCP network. Tests should avoid
+// bringing down this interface so they maintain spoke connectivity.
+func GetOCPInterfaceName(client *clients.Settings, nodeName string) (Name, error) {
```

Comment:
nit: Suggest to change this helper to GetEgressInterfaceName. At the time the helper was created in cnf-gotests, I didn't have enough knowledge to give it a better name..